projects/packages/publicize/src/class-publicize-base.php

```diff
@@ -589,17 +589,25 @@ public function get_display_name( $service_name, $connection ) {
 	 * @return string
 	 */
 	public function get_username( $service_name, $connection ) {
-		$cmeta = $this->get_connection_meta( $connection );
+		$cmeta    = $this->get_connection_meta( $connection );
+		$username = '';
```

Comment:
Since `username` field is deprecated in favour of `external_handle`, we don't need to update it. We are going to remove it anyway in the future.